test/ReflectionUtil.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Session;
+
+use ReflectionObject;
+
+final class ReflectionUtil
```

Comment:
Probably change that to a trait and implement in those tests which depend on that.